tests/foreman/api/test_docker.py

```diff
@@ -925,20 +1145,27 @@ def test_create_internal_docker_compute_resource(self, name):
         self.assertEqual(compute_resource.provider, DOCKER_PROVIDER)
         self.assertEqual(compute_resource.url, INTERNAL_DOCKER_URL)
 
-    @stubbed()
     @run_only_on('sat')
     def test_update_internal_docker_compute_resource(self):
-        """@Test: Create a Docker-based Compute Resource in the
-        Satellite 6 instance then edit its attributes.
+        """@Test: Create a Docker-based Compute Resource in the Satellite 6
+        instance then edit its attributes.
 
-        @Assert: Compute Resource can be created, listed and its
-        attributes can be updated.
+        @Assert: Compute Resource can be created, listed and its attributes can
+        be updated.
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        compute_resource = entities.DockerComputeResource(
+            name=gen_string('alpha'),
+            url=INTERNAL_DOCKER_URL,
+            organization=[self.org],
+        ).create()
+        self.assertEqual(compute_resource.url, INTERNAL_DOCKER_URL)
+        new_url = gen_url(scheme='http')
+        compute_resource.url = new_url
+        compute_resource = compute_resource.update(['url'])
+        self.assertEqual(compute_resource.url, new_url)
```

Comment:
Most NailGun methods do not mutate the objects they are called on. You can use this to write less dense code:

```python
        compute_resource.url = gen_url(scheme='http')
        self.assertEqual(
            compute_resource.url,
            compute_resource.update(['url']).url,
        )
```

Non-blocker comment.